controllers/base_controller.js

```diff
@@ -186,6 +224,15 @@ module.exports = function BaseControllerModule(pb) {
     };
 
     /**
+     * @method getTemplateService
+     * @param {Object} options for TemplateService
+     * @return {Object} TemplateService
+     */
+    BaseController.prototype.getTemplateService = function(tsOps) {
+        return new pb.TemplateService(tsOps);
+    };
```

Comment:
What was the need for this function?  Each controller already has an instance of TemplateService created during initialization.  Additional instances can be created by calling ```this.ts.getChildInstance()```.  The function will return a new instance and provide the same local registrations that the parent instance contained.

Was it just for mocking purposes?